examples/06_manipulating_images/plot_nifti_simple.py

```diff
@@ -49,7 +49,7 @@
 
 # %%
 # Visualize the mask using the 'generate_report' method
-# This report can be displayed in a Jupyter Notebook,
+# This report can be displayed in a Notebook,
 # opened in-browser using the .open_in_browser() method,
 # or saved to a file using the .save_as_html(output_filepath) method.
```

Comment:
With this PR it means that no report is ever saved in any example.

I think that we should have a systematic message that explains how to do this.